Frontend/src/components/salon-components/salonServices.tsx

```
@@ -6,29 +6,26 @@ import type { ColumnsType } from "antd/es/table";
 import { useStyles } from "../../app/salon/bookings/style/styles";
 import "@ant-design/v5-patch-for-react-19";
 import { ISalonService } from "@/providers/salonService-provider/context";
-import { useSalonServiceActions, useSalonServiceState } from "@/providers/salonService-provider";
-
-
+import {
+  useSalonServiceActions,
+  useSalonServiceState,
+} from "@/providers/salonService-provider";
 
 const SalonServices = ({}: { salonServices?: ISalonService[] }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)